examples/cdi-qualifier/README_pt.adoc

```diff
@@ -0,0 +1,99 @@
+= CDI @Qualifier
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+== Introdução
+
+As vezes nós need escrever várias implementações para uma interface de regras de negócios, para aumentar o desacoplamento vamos injetar apenas a interface e o CDI vai escolher a implementação correta, para ajudar o CDI com esta escolha nós criamos os **qualificadores**.
```

Comment:
in the sentence "nós need escrever", i think that you forgot to translate =)

Reply:
done.